docs/markdown/Reference-manual.md

```diff
@@ -741,6 +743,7 @@ This object is returned by [`meson.get_compiler(lang)`](#meson-object). It repre
 
 - `get_id()` returns a string identifying the compiler (e.g. `'gcc'`)
 - `version()` returns the compiler's version number as a string
+- `cmd_array()` returns the command to invoke the compiler as a list (e.g. `['ccache', 'cc']`), available since 0.24.0
```

Comment:
In this case, this feature is used by systemd for an edge-case which can't be done in any other way, and such edge-cases will always exist, IMO.